drivers/wireless/gs2200m.c

```diff
@@ -865,19 +865,19 @@ static uint16_t _read_data_len(FAR struct gs2200m_dev_s *dev)
 
   _write_data(dev, hdr, sizeof(hdr));
 
+  /* NOTE: busy wait 30us
+   * workaround to avoid an invalid frame response
+   */
+
+  up_udelay(30);
```

Comment:
Can we use a sleep instead to not busy spin, or is this too fast?

Reply:
>Can we use a sleep instead to not busy spin, or is this too fast?

@btashton 
Currently, Spresense uses a 10ms tick, so if we replace it with a sleep function, performance would be very bad.